contrib/jemalloc/include/jemalloc/internal/sc.h

```diff
@@ -169,7 +169,7 @@
  * Size class N + (1 << SC_LG_NGROUP) twice the size of size class N.
  */
 #define SC_LG_NGROUP 2
-#define SC_LG_TINY_MIN 3
+#define SC_LG_TINY_MIN 4
```

Comment:
Looks like we at least need to gate this on !32-bit as it causes this breakage so I'll gate it on MRS for now. https://github.com/CTSRD-CHERI/cheribsd/pull/1898#issuecomment-1760194126

Reply:
I assume we don't think it'll make a difference for performance?

Reply:
It might increase memory use for rather odd programs and I guess there's some potential for cache effects. It seems to work for hybrid so we could enable it on !ILP32 if that seemed like a more balanced approach.

Reply:
Arguably it might be a "cost" of our temporal safety so for comparing against an aarch64 baseline I think you'd want the baseline to use the old value.

Reply:
I've gone with enabling it only for MRS. That makes it clear where the unavoidable cost lies.